cmd/minikube/cmd/delete.go

```diff
@@ -725,3 +738,46 @@ func getPids(path string) ([]int, error) {
 
 	return pids, nil
 }
+
+// deleteKnownHosts reads minikube nodes' keys from the minikube home folder
+// and removes them from the known_hosts file. This is the long term solution
+// for issue https://github.com/kubernetes/minikube/issues/16868
+func deleteKnownHosts() {
+	// remove this line from known_hosts file if it exists
+	knownHosts := filepath.Join(homedir.HomeDir(), ".ssh", "known_hosts")
+	machinesDir := filepath.Join(localpath.MiniPath(), "machines")
+	fileInfo, err := os.ReadDir(machinesDir)
+	if err != nil {
+		klog.Warningf("error reading machines in minikube home: %v", err)
+		return
+	}
+	if _, err := os.Stat(knownHosts); err == nil {
```

Comment:
```suggestion
	if _, err := os.Stat(knownHosts); err != nil {
```
We're returning if the `known_hosts` file is found, we want to opposite of this.